src/pages/ProfilePage.tsx

```diff
@@ -0,0 +1,65 @@
+import Sidebar from "../components/Sidebar";
+import "../index.css";
+import React from 'react';
+import ReactDOM from 'react-dom/client';
```

Comment:
```suggestion
import React from 'react';
import ReactDOM from 'react-dom/client';
```
Remove unused imports